lineapy/db/db.py

```diff
@@ -665,7 +669,11 @@ def delete_node_value_from_db(
             .first()
         )
         if value_orm is None:
-            track(ExceptionEvent("UserException", "Value node not found"))
+            track(
+                ExceptionEvent(
+                    ErrorType.DATABASE, "Value not found for the node"
```

Comment:
same as above, clarify that this is value path

Reply:
Retaining the original text in this case because it is in the context of querying node ORM